apis/certificateregistry/certificateregistryapi.yaml

```diff
@@ -0,0 +1,376 @@
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          description: 'To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: 'OK ! Operation successful."Certificate download" operation was executed successfully.'
+          schema:
+            allOf:
+              - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+              - $ref: '#/definitions/CertificateDownloadResponse'
+        '400':
+          description: 'BAD REQUEST. The "Certificate generation" operation failed. You may have missed providing input for a mandatory parameter.'
+          schema:
+            $ref: '#/definitions/IssueCertificateErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR ! Looks like something went wrong. These errors are tracked automatically, but if the problem persists, contact us. In the meantime, try refreshing.'
+  /certreg/v1/certs/validate:
+    post:
+      description:  >-
+        This API is associated with validating digitally generated certificates.
+           - The **_/certreg/v1/certs/validate_** endpoint executes the "validate certificate" request based on parameters provided in the request body.
+           - All fields marked with an * are mandatory
+           - Required fields cannot be null or empty
+      summary: validation of certificate
+      tags:
+        - Validate Certificate Api
+      operationId: validate certificate
+      produces:
+        - application/json
+      parameters:
+        - name: Body
+          in: body
+          required: true
+          description: >-
+            The body refers to the format of the request. The body contains metadata about the request to validate the certificate.
+          schema:
+            allOf :
+              - $ref: '#/definitions/CertificateValidateRequest'
+        - name: ts
+          in: header
+          required: true
+          type: string
+          description: 'The timestamp at which the **validating certificate** request was sent. '
+        - name: X-msgid
+          in: header
+          required: true
+          type: string
+          description: 'A unique ID that identifies the request in case the same API is executed multiple times.'
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          description: 'To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: 'OK ! Operation successful."Certificate generation" operation was executed successfully.'
```

Comment:
This is for validate , so change description.

Reply:
done